src/public-installer.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+cd "$(dirname "$0")"
+
+## This is the public installer script that will fetch the kubernetes installer run it
+## The intention is that the AWS install script pieces will one day be cherry-picked into here.
```

Comment:
Since this is public facing, we should take out these notes.

Reply:
kk